Client/Frontend/UserContent/UserScripts/MainFrame/AtDocumentStart/BraveGetUA.js

```diff
@@ -0,0 +1,22 @@
+/* vim: set ts=2 sts=2 sw=2 et tw=80: */
```

Comment:
wow a vim magician!
Want to keep this line?

Reply:
Lol! I'll remove it 😂

Reply:
This is actually on all of the .js files, so just going to leave 🤷‍♀

Reply:
fine by me